lib/logstash/outputs/email.rb

```diff
@@ -119,7 +119,8 @@ def register
           :password             => options.fetch("password", nil),
           :authentication       => options.fetch("authenticationType", nil),
           :enable_starttls_auto => options.fetch("starttls", false),
-          :debug                => options.fetch("debug", false)
+          :debug                => options.fetch("debug", false),
+          :openssl_verify_mode  => options.fetch("openssl_verify_mode", 'peer')
```

Comment:
I wonder what happen when TLS/SSL is not enabled if we setup this option like this ? can you test that?